vue/components/form/inputs/datetime.vue

```diff
@@ -17,13 +17,8 @@
         @input="dateTimeInput"
       />
       <div style="display: inline-block; float: left; width: 15%;">
-        <div
-          type="button"
-          style="border: none; justify-content: center;"
-          @click="clear"
-        >
-          <IonIcon class="datetime-delete-icon" name="trash-outline"/>
-        </div>
+        <InputClearButton :button-style="'border: none; justify-content: center;'"
```

Comment:
Tudi to mi ni preveč všeč: naredil si komponento, potem jo moraš pa takoj različno stilirat glede na to, v kakšnem containerju imaš uporabljeno. Ne bi bilo bolj smiselno, da komponento samo pustiš pri miru in postaviš container tako, da bo prikazana pravilno?

Reply:
done